regulations/settings/base.py

```diff
@@ -173,6 +173,15 @@
         'OPTIONS': {
             'MAX_ENTRIES': 1000,
         },
+    },
+    'regs_gov_cache': {
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'KEY_PREFIX': 'regs.gov',
+        'LOCATION': 'redis://localhost:6379/0',
+        'TIMEOUT': 60*60*24,
+        'OPTIONS': {
+            'IGNORE_EXCEPTIONS': True,
+        },
```

Comment:
Can we move this config into `notice-and-comment` and rely on a dummy/filebased/memory cache in -site? Generally, we want to assume as little as possible of folks running -site.